modules/nf-core/biohansel/meta.yml

```diff
@@ -0,0 +1,58 @@
+name: "biohansel"
+description: write your description here
```

Comment:
Please write a description here :)

Reply:
nice catch, brain wrote it, fingers did not

Reply:
happens to the best of us!